docker/tidb-sysbench/release.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TAG="${1:-"latest"}"
+docker tag "pingcap/tidb-sysbench:$TAG" "gcr.io/pingcap-public/pingcap/tidb-sysbench:$TAG"
+gcloud docker -- push "gcr.io/pingcap-public/pingcap/tidb-sysbench:$TAG"
```

Comment:
We should also publish the image to DockerHub

Reply:
I don't have permission for that. Perhaps I can get permission to push just this image? But I am wondering if between this and uhub we should have some ability to move images between registries.